packages/perseus/src/components/math-input.tsx

```diff
@@ -67,6 +68,13 @@ type Props = {
     analytics: PerseusDependenciesV2["analytics"];
 };
 
+type InnerProps = Props & {
+    // NOTE(john): We'd like to use the real MathInputStrings type here, but
+    // getting the types and imports to work correctly turns out to be really
+    // hard, it's not worth it as we are just passing the types through.
+    mathInputStrings: any;
```

Comment:
Should this comment be referencing "math input strings" here or "perseus strings?"

Reply:
This is correct - it should be math input strings. This component is confusing as it needs access to both the math input strings AND the perseus strings!